### PR TITLE
ISPN-8706 Ignore HotRodMergeTest

### PR DESCRIPTION
**Pull request by ryanemerson**

### CODE REVIEW

File: server/hotrod/src/test/java/org/infinispan/server/hotrod/HotRodMergeTest.java [hunk 1]
```diff
@@ -27,7 +27,7 @@
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
-@Test(groups = "functional", testName = "server.hotrod.HotRodMergeTest")
+@Test(groups = "functional", testName = "server.hotrod.HotRodMergeTest", enabled = false)
```
Comment (wburns) [inline server/hotrod/src/test/java/org/infinispan/server/hotrod/HotRodMergeTest.java]: I wonder should we just add it to the unstable group instead? I am not up to speed on how we are disabling nowadays :)

Comment (danberindei) [reply server/hotrod/src/test/java/org/infinispan/server/hotrod/HotRodMergeTest.java]: Yeah, we normally add it to the `unstable` group, e.g. `groups = {"functional", "unstable"}`

Comment (ryanemerson) [reply server/hotrod/src/test/java/org/infinispan/server/hotrod/HotRodMergeTest.java]: I didn't realise, good to know. I'll change it

**Comment by gustavonalle** [comment]
merged thanks!

**Pull request merged**